online_store/gt_sport_v2.js

```diff
@@ -87,21 +92,22 @@ function productDescriptionDisplay() {
       <p> <button type="button" id="addToShoppingCartBtn">Add to Shopping Cart</button>
       `;
     productDescription.appendChild(descriptions);
-  };
-};
+  }
+}
 productDescriptionDisplay();
 
 function addProductsToCart() {
-  for (let i = 0; i < sportProducts.length; i++) {
-    shoppingCart.push(sportProducts[i]);
-  };
+  shoppingCart.push(sportProducts[i]);
```

Comment:
you can not use **i** before it is being declared.

Reply:
OK, changed.